twisted/twisted
# #11573 Remove Trac documentation build tools

## Scope and purpose

Fixes #11573

We no longer use twisted-web to host the doccs or Trac theme to integrate the docs to trac theme.
We also no longer build the documentation to be distributed on a separate site.
We only have Read The Docs.

This is cleanup for the code that is no longer used.

The docs are now hosted on Read The docs with build defined in `.readthedocs.yml` and 'docs/conf.py`

The docs are generated for dev using tox via `tox -e narrativedocs` that calls sphinx-build directly.

I think that by removing the script, it should be easier to understand how the docs are generated.

We no longer have some automated tests, but at the same time, we no longer have custom Sphinx theme or custom extensions that we need to mantain.


## How to test

* Check that you can build the docs locally via `tox -e narrativedocs`
* Check the Read The Docs version at https://twisted--11576.org.readthedocs.build/en/11576/

## Contributor Checklist:

* [x] I ran `tox -e lint` to format my patch to meet the [Twisted Coding Standard](https://docs.twistedmatrix.com/en/stable/core/development/policy/coding-standard.html)
* [x] I have created a newsfragment in src/twisted/newsfragments/ (see: [News files](https://github.com/twisted/trac-wiki-archive/blob/trunk//ReviewProcess.mediawiki#News_files))
* [x] The title of the PR starts with the associated issue number (without the `#` character).
* [NA] I have updated the automated tests and checked that all checks for the PR are green.
* [x] The merge commit will use the below format
  The first line is automatically generated by GitHub based on PR ID and branch name.
  The other lines generated by GitHub should be replaced.

```
Merge pull request #123 from twisted/4356-branch-name-with-issue-id

Author: adiroiban
Fixes #11573

Remove tools used to use documentation for Trac integration.
```
